docs/sources/shared/configuration.md

```diff
@@ -3557,10 +3557,23 @@ The `frontend_worker` configures the worker - running within the Loki querier -
 # CLI flag: -querier.id
 [id: <string> | default = ""]
 
-# The grpc_client block configures the gRPC client used to communicate between a
-# client and server component in Loki.
+# Configures the gRPC client used to communicate between the querier and the
```

Comment:
I'd reword this the other way around: first what;s the common use-case, later the note about use_separated_clients being true.